import.sql

```diff
@@ -0,0 +1,87 @@
+CREATE TEMP TABLE data_import (
```

Comment:
Not sure if project root really is a good place for this file :/

Reply:
moved to `useful scripts/`